README.md

```diff
@@ -1,17 +1,23 @@
-# lightsheet
+# Lightsheet
+A lightweight spreadsheet implementation with minimal dependencies written in Typescript. Lightsheet implements a standalone core for efficient spreadsheet data management, as well as a simple UI component for displaying data. Lightsheet supports displaying multiple sheets at once with cross-referenced data. 
+
+The API has been designed with extensibility in mind, implementing features such as custom function definitions and an open event system. The API is documented in `main.ts`, and core structures are described in the project wiki.
```

Comment:
The api will be documened here is this readme, and core functionlity is documented in wiki. and the main.ts comments are for code readability and intelisence, I wouldnt call it documentation.

Reply:
Are we planning on documenting the API further than what we'll include in `main.ts`? When we discussed this on Wednesday, I understood we may not have time for it. Either way, this readme reflects the current situation and can be updated later.